procedural/timeline/src/main/kotlin/gov/nasa/ammos/aerie/procedural/timeline/util/duration/Duration.kt

```diff
@@ -33,5 +33,8 @@ operator fun Duration.rem(divisor: Duration): Duration = this.remainderOf(diviso
 operator fun Instant.plus(duration: Duration): Instant = plusMillis(duration / MILLISECOND)
     .plusNanos(1000 * ((duration % MILLISECOND) / MICROSECOND))
 
-/** Subtracts a duration from an instant, to produce another instant. */
+/** Subtracts an instant from another instant, represented as a duration. */
 operator fun Instant.minus(other: Instant): Duration = microseconds(other.until(this, ChronoUnit.MICROS))
+
+/** Subtracts a duration from an instant, to produce another instant. */
+operator fun Instant.minus(duration: Duration) = this.plus(-duration)
```

Comment:
Probably unnecessary due to Kotlin's type inference, but it matches the explicitness of your doc comment.

```suggestion
operator fun Instant.minus(duration: Duration): Instant = this.plus(-duration)
```